Automattic/jetpack
# Subscription Form: change the  default placeholder and button text values

The default values for the Subscription Form block's email field placeholder and submit button text do not match the values shown on the site.

Fixes #18072

#### Changes proposed in this Pull Request:
* Match the default `subscribePlaceholder` and `submitButtonText` default values to the default values used on the backend:
    * `subscribePlaceholder` = `Email Address`
    * `submitButtonText` = `Subscribe`


#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install the stable version of Jetpack. Activate and connect Jetpack.
2. Navigate to wp-admin -> Jetpack Settings -> Discussion.
3. Scroll down to the Subscriptions card and enable the `Let visitors subscribe to new posts and comments via email setting`.

##### Test with an existing block
4. Create a new post.
5. Add a Subscription Form block.
6. Navigate to the post. Notice that the placeholder and button text do not match the text in the post editor, as described in #18072.
7. Checkout this branch.
8. Navigate to the post editor and verify that no errors occur. The placeholder text should now be `Email Address`, and the button text should now be `Subscribe`.
9. Navigate to the post and verify that no errors occur. 

##### Test with a new block
10. Create a new post.
11. Add a Subscription Form block. The placeholder text should now be `Email Address`, and the button text should now be `Subscribe`.
12. Navigate to the post and verify that no errors occur. The placeholder text should  be `Email Address`, and the button text should be `Subscribe`.


#### Proposed changelog entry for your changes:


* Subscription Block: display the correct default placeholder and button text strings in the post editor.
